src/components/progress-circle/progress-circle.ts

```diff
@@ -251,8 +251,8 @@ function clamp(v: number) {
  * Returns the current timestamp either based on the performance global or a date object.
  */
 function now() {
-  if (typeof performance !== 'undefined' && performance.now) {
-    return performance.now();
+  if (typeof window.performance !== 'undefined' && window.performance.now) {
```

Comment:
Probably don't need the `typeof` check anymore.  Could simplify to `if (window.performance && window.performance.now)`